src/checkIfConditionMatches.js

```diff
@@ -59,10 +59,8 @@ export function checkIfConditionMatchesProperties(node, formValues, formFields,
 
     return validateFieldSchema(
```

Comment:
> [@brennj] @sandrina-p can i get you to approve these changes when you have a chance please? :)

Do you need it today or can it be Monday morning?

Reply:
Whenever works ;)